ang/actions/ReturnValue.html

```diff
@@ -0,0 +1,45 @@
+<div class="crm-accordion-wrapper crm-sqltask-php collapsed relativePosition">
+    <div class="crm-accordion-header accordion-header-custom active action-label">
+        <label class="input-checkbox">
+            <input ng-true-value="'1'" style="cursor: pointer;" ng-false-value="'0'" ng-model="model['enabled']" type="checkbox">
+            &nbsp;
+            {{ts('Return Value')}}
+        </label>
+        <span ng-if="model['action_title']" class="sql-task-action-title">{{model['action_title']}}</span>
+        <label class="header-delete-btn" ng-click="removeItemFromArray(index)">
+            <i class="crm-i fa-trash header-delete"></i>
+            {{ts('Delete')}}
+        </label>
+    </div>
+    <div class="crm-accordion-body" style="display: none;">
+        <div class="sql-task-action-template-wrapper">
+            <action-template-select
+                model='model'
+                action-templates='actionTemplates'
+                action-template='actionTemplate'>
+            </action-template-select>
+            <div class="clear"></div>
+            <action-template-form
+                model='model'
+                action-templates='actionTemplates'
+                action-template='actionTemplate'>
+            </action-template-form>
+        </div>
+        <div class="sql-tasks">
+            <action-additional-info model='model' fieldPrefix="'return_value'" index='index'></action-additional-info>
+
+            <text-input isrequired="getBooleanFromNumber(model['enabled'])"
+                        model="model['table']" fieldlabel="ts('Data Table')" fieldid="'return_value_table' + index"
+                        sizelength="60" inputMaxWidth="'500px'"
+                        showHelpIcon=true
+                        helpaction='onInfoPress("Data Source", "id-return-value-table", "CRM\/Sqltasks\/Action\/ReturnValue")'>
```

Comment:
You can add a `.hlp` file for this action in `templates/CRM/Sqltasks/Action` to describe what to do with Data Table and Parameter.